### PR TITLE
[TE] rootcause legend styling

### PR DESCRIPTION
**Pull request by justYves**
### What's new: 
- styled rootcause legend component
- added tooltip for remove and view toggling

### Screenshots: 
(I know there is a typo)
screen shot 2017-12-07 at 3 48 21 pm
screen shot 2017-12-07 at 3 48 30 pm
screen shot 2017-12-07 at 3 48 35 pm

### CODE REVIEW

File: thirdeye/thirdeye-frontend/app/pods/components/rootcause-chart/component.js [hunk 1]
```diff
@@ -31,6 +32,9 @@ export default Ember.Component.extend({
     show: false
   },
 
+  /**
+   * Adding the buildTooltip Template helper to the this context
```
Comment (ttbach) [inline thirdeye/thirdeye-frontend/app/pods/components/rootcause-chart/component.js]: nit: "Adding...to **the** **this** context" (typo?)

File: thirdeye/thirdeye-frontend/app/pods/components/rootcause-legend/component.js [hunk 2]
```diff
@@ -33,12 +35,25 @@ export default Ember.Component.extend({
     }
   ),
 
+  /**
+   * Parses the validUrns and builds out 
+   * a Mapping of event Types to a mapping of urns
+   */
```
Comment (ttbach) [inline thirdeye/thirdeye-frontend/app/pods/components/rootcause-legend/component.js]: `@type`?

File: thirdeye/thirdeye-frontend/app/pods/components/rootcause-legend/template.hbs [hunk 3]
```diff
@@ -5,39 +5,46 @@
...
+          </div>
+          <a class="rootcause-legend__remove-icon" {{action "removeUrn" urn}}>
+            <i class="glyphicon glyphicon-remove"></i>
+            {{tooltip-on-element
+              text='Remove event'}}
+          </a>
+        </li>
+      {{/each-in}}
+    </ul>
+  {{/each-in}}
```
Comment (ttbach) [inline thirdeye/thirdeye-frontend/app/pods/components/rootcause-legend/template.hbs]: Missing EOL

File: thirdeye/thirdeye-frontend/app/pods/components/rootcause-chart/component.js (hunk 1 above)
Comment (justYves) [reply thirdeye/thirdeye-frontend/app/pods/components/rootcause-chart/component.js]: Sorry for the confusion, I meant : => this.buildTooltip.
The tooltip callback function doesn't have access to the global context, but has access to the `this`, I'm setting it on the `this` context

**Pull request merged**